targets/unity-v2/source/ExampleTestProject/Assets/PlayFabSDK/Shared/Internal/PlayFabHttp/PlayFabHTTP.cs

```diff
@@ -446,7 +446,7 @@ protected internal static void SendEvent(string apiEndpoint, PlayFabRequestCommo
             }
         }
 
-        protected internal static void ClearAllEvents()
+        public static void ClearAllEvents()
         {
```

Comment:
If this is no longer internal, should the file (or just this function) be moved to a more appropriate folder location name?

Reply:
Nope! This file is still under the PlayFab.Internal name space so this should be fine #resolved